### PR TITLE
Updated gem to latest Mongoid and Yard versions

### PR DESCRIPTION
**Pull request by kalinon**
I have updated and tested this with the newer Yard 9.0+ and Mongoid 5.0+ versions.

### CODE REVIEW

**Review by alsemyonov — CHANGES_REQUESTED** [review]
Instead of forcing users to update to latest versions of `yard` and `mongoid`, we need to loosen gem dependencies to include both old and new version of gems. We should not drop support of old gem versions.

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ gemspec
 
 group :development do
   gem 'rake'
-  gem 'mongoid', '~> 3.0'
+  gem 'mongoid', '>= 5.0'
```
Comment (kalinon) [inline Gemfile]: Thanks for the comments, I'm new to the whole gem thing. That makes sense tho

File: yard-mongoid.gemspec [hunk 2]
```diff
@@ -19,5 +19,5 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = %w(lib)
 
-  s.add_dependency 'yard', '~> 0.8.4'
+  s.add_dependency 'yard', '~> 0.9'
```
Comment (kalinon) [inline yard-mongoid.gemspec]: Point taken! Thanks!

**Comment by kalinon** [comment]
@alsemyonov i have made your suggested changes. Thanks for the feedback!

**Comment by alsemyonov** [comment]
@kalinon, could you remove version bump from this branch and squash other changes into one commit? 
Then I’m ready to merge this PR.

**Comment by kalinon** [comment]
@alsemyonov this is done!

**Review by alsemyonov — APPROVED** [review]

**Comment by alsemyonov** [comment]
@kalinon, I have released version 0.1.0. Thank you for your contribution!

**Pull request merged**